core/services/chainlink/application.go

```diff
@@ -103,6 +103,8 @@ type Application interface {
 
 	// ID is unique to this particular application instance
 	ID() uuid.UUID
+
+	GetSecretGenerator() SecretGenerator
```

Comment:
Get/Set prefixes can generally be omitted in Go, although the local style here is aleady a mix so :shrug: 
```suggestion
	SecretGenerator() SecretGenerator
```